packages/next-auth/src/core/types.ts

```diff
@@ -362,6 +362,7 @@ export interface CookiesOptions {
   pkceCodeVerifier: CookieOption
   state: CookieOption
   nonce: CookieOption
+  generateSessionToken: () => string
```

Comment:
This `CookiesOptions` is more like for cookies-related configurations, as you can see they are all `CookieOption`
We should move this new config to somewhere else. I suggest it be inside of `session`.

Reply:
Ah good suggestion! Made the changes.